portal/src/texts/profile/picture.md

```diff
@@ -1,5 +1,288 @@
+
+[bilde: riktig + med feil skilting og dashboard]
+
+	•	Gjenkjennelige omgivelser
+	•	Vi viser mangfoldet i det norske samfunnet
+	•	Varierte omgivelser
+	•	Vi skifter gjerne bilder med årstidene og høytider
+	•	Se opp for fremmedgjørende elementer
+
+## Kategoribilder
+
+Når vi snakker om et konkret forsikringsprodukt kan det være vanskelig å finne bilder som passer perfekt. Her er noen tips og retningslinjer til de ulike forsikringene. Felles for kategoribildene er at vi ønsker fokus på en situasjon som oppstår rundt eller i samspill med produktet fremfor produktet i seg selv.
+
+Skadeforsikring:
+Vi viser folk og tingene deres i samspill heller enn å ta bilde av et produkt. Viser vi situasjoner der skader oppstår?
+
+Bilforsikring:
+Vi tar bilder i og rundt bilen. I trafikken, ved av-og-pålessing av folk og ting eller stell av bilen. For eksempel pakking til ferie, henting på trening, handletur eller bilvask. Situasjonen er viktigere enn bilen, det skal ikke oppfattes som reklame for en bestemt biltype. Vær oppmerksom på at bilen skal kjøre på norske veier.
+
+[bilde: i bilen + ved bilen]
+
+Resieforsikring:
+Når det gjelder reiseforsikring kan vi bruke bilder fra utlandet, men da gjerne i typiske situasjoner man havner i som turist. Her kan vi appellere til ulike typer som den eventyrlystne, storbyturisten, solslikkeren, eller lokalturisten.
+
+Obs: Fremtind er et bærekraftig selskap og vi ønsker ikke å bruke bilder av fly, cruiseskip eller aktiviteter som er ødeleggende for natur eller lokalsamfunn.
+
+[bilde: generell turist + bilde av sted/reisetyper vi ikke “reklamerer” for]
+ 
+Dyreforsikring:
+Dyr har sin egen personlighet og det er ok å la de eie bildet alene, men det er fint også her å vise samspillet mellom dyr og mennesker. Dyr egner seg fint til bilder tatt “rett ovenfra”.
+
+[bilde: dyr ovenfra]
+
+	•	Vi skreddersydder bilder til de ulike forsikringsproduktene
+	•	Skade
+	•	Vi tar biler rundt og i bilen uten at det ser ut som en bilreklame
+	•	Vi bruker typiske turistsituasjoner for reiseforsikring
+	•	Dyreforsikringsbilder kan være dyr alene eller med folk
+
+## Lys og farger
+Vi skaper en inviterende og virkelighetsnær atmosfære ved å bruke dagslys eller naturlig lyssetting og fargekorrigering.
+
+[bilde: prakteksemplar]
+
+Vi unngår skarpt lys og harde skygger, spesielt i ansikter.
+
+[bilde: skygger i ansikt, bilde der hovedmotivet forsvinner i skygge]
+
+Unngå hvit bakgrunn som skaper høy kontrast og skarpe kanter. Dette oppstår ofte der det er hvitmalte vegger eller ute med lys himmel.
+
+[bilde: bilde med overeksponert himmel + riktig eksponering eller annet vær + innebilde med ulik veggfarge]
+
+Vi bruker ikke effekter, filtre eller overflødig bruk av refleksjoner (lens flare) på bildene våre. Alle bildene i Fremtinds bildedatabase er redigert og fargekorrigert så de oppfyller kravene i bildestilen og oppleves like.
+
```

Comment:
Vise eksempler på for lite og for mye korrigering – de fleste digitale bilder har behov for korrigering. Hva er lov/ikke lov? 

(Unngå bilder tatt i skarpt sollys mht skygger? Er det mulig å få til bilder som kun er tatt i golden hour? )
(Steinop)